src/main/java/org/capstone/job_fair/services/interfaces/company/CompanyService.java

```diff
@@ -16,4 +19,5 @@
     Boolean deleteCompany(String id);
     Integer getCountById(String id);
     Optional<CompanyEntity> findCompanyById(String id);
+    void registerAJobFair(CompanyRegistrationDTO company, List<CompanyJobFairRegistrationRequest.JobPosition> jobPositions);
```

Comment:
createCompanyRegistration is a better name?? Why CompanyRegistrationDTO name is company? Why using request here? The request should be mapping to DTO list.